core/src/main/scala/sttp/client3/internal/ToCurlConverter.scala

```diff
@@ -6,10 +6,20 @@ import sttp.client3.NoBody
 import sttp.model.MediaType
 
 class ToCurlConverter[R <: RequestT[Identity, _, _]] {
-  def apply(request: R): String = {
-    val params = List(extractMethod(_), extractUrl(_), extractHeaders(_), extractBody(_), extractOptions(_))
+
+  def apply(request: R): String = create(request, sensitiveHeaders = None)
```

Comment:
I'd understand `sensitiveHeaders = None` as "no sensitive headers" while it means "default sensitive headers". Maybe we can just a default value of `sttp.model.HeaderNames.SensitiveHeaders` instead of an `Option[Set[String]]`?

Reply:
done